### PR TITLE
Add engines to OpenAPI

### PR DESCRIPTION
**Pull request by tyrannosaurus-becks**
Noticed we've added a couple of engines that it would be nice to have in the `gen-openapi` script.

### CODE REVIEW

File: scripts/gen_openapi.sh [hunk 1]
```diff
@@ -36,7 +36,9 @@ vault auth enable github
 vault auth enable jwt
 vault auth enable kubernetes
 vault auth enable ldap
+vault auth enable oidc
```
Comment (kalafut) [inline scripts/gen_openapi.sh]: Wondering if we need this one since its just a name alias for jwt?

Comment (tyrannosaurus-becks) [reply scripts/gen_openapi.sh]: Ah! OK, I will strike it.

**Comment by jefferai** [comment]
Does this need to be run and have the output stored anywhere or anything?

**Comment by kalafut** [comment]
This is mainly for development use now, providing an easy aggregate of all OpenAPI output. The script isn't invoked by anything automatically, nor is the output checked in.

**Pull request merged**